Adamant/Modules/Chat/View/Subviews/BadgeView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  BadgeView.swift
+//  Adamant
+//
+//  Created by Аркадий Торвальдс on 26.08.2024.
+//  Copyright © 2024 Adamant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class BadgeView: UIView {
```

Comment:
final

Reply:
are you sure we need a separate `BadgeView` class? It seems like all we can do with `label`.

Reply:
I think you're right, I can just use UILAbel